elasticdl/docker/Dockerfile.dev

```diff
@@ -24,5 +24,10 @@ RUN pip install -r /requirements-dev.txt --extra-index-url=${EXTRA_PYPI_INDEX}
 # rest part of ElasticDL.  The generated data will be in /data.
 COPY elasticdl/python/data/recordio_gen/image_label.py /var/image_label.py
 RUN python /var/image_label.py --dataset mnist --fraction 0.25 \
-	--records_per_shard 4096 /data && \
-    rm -rf /root/.keras/datasets
+	--records_per_shard 4096 /data
+
+# Copy frappe dataset
+COPY elasticdl/python/data/recordio_gen/frappe_recordio_gen.py /var/frappe_recordio_gen.py
+RUN python /var/frappe_recordio_gen.py --data /root/.keras/datasets --output_dir /data/frappe \
```

Comment:
After building the new docker image, we won't need to download the dataset each time? It seems that a unit test fails due to the error of downloading a dataset for github.

Reply:
Yes, I have cache the dataset in travis CI. But, locally, it has to downloading the dataset at the first time.

I just follow the former code here. If we remove the directory, the size of this image will be smaller.

And for the travis, at the first time, the dataset will download inside docker. Since we mount the host directory to docker container, the dataset will also be write to host disk. Then, we cache the dataset in host. At the second time, the dataset will be used directly, which speed up the ci.